src/mame/video/namco_c355spr.h

```diff
@@ -41,7 +41,10 @@ class namco_c355spr_device : public device_t, public device_gfx_interface, publi
 	void get_sprites(const rectangle cliprect);
 	void copy_sprites(const rectangle cliprect);
 
+	void clear_screen_bitmap() { m_screenbitmap.fill(0xffff); }
+	void clear_screen_bitmap(const rectangle cliprect) { m_screenbitmap.fill(0xffff, cliprect); }
```

Comment:
Are these functions supposed to be hooked up somewhere in the drivers?  Right now they aren’t actually doing anything.

Reply:
When I did the regression for that day - yes - they seem to be resolved or rather reverted to previous states.